org.lflang/src/org/lflang/federated/extensions/CExtension.java

```diff
@@ -611,7 +611,7 @@ private String generateCodeToInitializeFederate(FederateInstance federate, Linke
         code.pr(String.join("\n",
                             "// Initialize the socket mutex",
                             "lf_mutex_init(&outbound_socket_mutex);",
-                            "lf_cond_init(&port_status_changed);"
+                            "lf_cond_init(&port_status_changed, &outbound_socket_mutex);"
```

Comment:
I don't know what's going on here, but that's probably because I haven't yet checked out your reactor-c PR...